packages/example/src/Provider.tsx

```diff
@@ -0,0 +1,91 @@
+import * as React from 'react';
+import Card from "react-bootstrap/Card";
+import Button from "react-bootstrap/Button";
+import Badge from "react-bootstrap/Badge";
+
+import { TrustProvider } from "@trustwallet/provider";
+
+interface IState {
+  getAccountsLoading: boolean,
+  signTransactionLoading: boolean
+}
+
+interface IProps {
+  openModal(title: string, body: React.ReactNode): void,
+  closeModal(): void,
+}
+
+export default class Provider extends React.Component<IProps, IState> {
+  constructor(props: IProps) {
+    super(props);
+    this.state = {
+      getAccountsLoading: false,
+      signTransactionLoading: false
+    };
+  }
+
+  testGetAccounts = async () => {
+    let { openModal } = this.props;
+
+    try {
+      let accounts = await TrustProvider.getAccounts();
+      openModal("Accounts", <samp>{JSON.stringify(accounts)}</samp>);
+    } catch (e) {
+      openModal("Accounts Error", <div>{e}</div>);
+    }
+  };
+
+  testSignTransaction = async () => {
+    let { openModal } = this.props;
+
+    try {
+      let network = 118; // Cosmos
+      let transaction = {
+          "accountNumber": "1035",
+          "chainId": "cosmoshub-2",
+          "fee": {
+            "amounts": [
+              {
```

Comment:
can you add example for staking and unstake ?

Reply:
Yep. I can